test/core/utils/is-hidden.js

```diff
@@ -1,3 +1,19 @@
+function createContentHidden() {
+	'use strict';
+	var group = document.createElement('div');
+	group.innerHTML = '<label id="mylabel">Label</label><input aria-labelledby="mylabel" type="text" />';
+	return group;
+}
+
+function makeShadowTreeHidden(node) {
```

Comment:
What makes this tree hidden? I don't see any special CSS that would hide it, or a closed mode.